definitions/npm/@storybook/react_v3.x.x/flow_v0.28.x-/react_v3.x.x.js

```diff
@@ -1,13 +1,14 @@
 type NodeModule = typeof module;
 
 declare module '@storybook/react' {
-  declare type Renderable = React$Element<any>;
-  declare type RenderFunction = () => Renderable;
+  declare type Renderable = React$Node;
```

Comment:
`React$Node` is only available from Flow `>=0.53.0`, so this breaks all previous versions. 
The libdef should be cloned and tweaked for later versions:
`definitions/npm/@storybook/react_v3.x.x/flow_v0.53.x-/react_v3.x.x.js `

Reply:
I would say to revert it for the `flow_v0.28.x-v0.52.x` and can stay as is for the `flow_v0.53.x`.